extensions/modules/file/src/test/xquery/modules/file/serialize.xqm

```diff
@@ -23,40 +23,126 @@ xquery version "3.1";
 
 module namespace serialize="http://exist-db.org/testsuite/modules/file/serialize";
 
-import module namespace test="http://exist-db.org/xquery/xqsuite" at "resource:org/exist/xquery/lib/xqsuite/xqsuite.xql";
 import module namespace file="http://exist-db.org/xquery/file";
+import module namespace helper="http://exist-db.org/xquery/test/util/helper" at "resource:util/helper.xqm";
```

Comment:
Why are we using `resource:` here?

Reply:
As discussed in the community call this is necessary for the import to work.

Reply:
As discussed on the Community Call.. I will take a look into this to find out where the issue is.

Reply:
Which will not block this PR getting merged right?
This method of importing modules is used in multiple places throughout the codebase.
I would prefer you open an issue and not discuss this here.

Reply:
@adamretter ?